kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/attachmentview/KmDocumentView.java

```diff
@@ -349,7 +349,13 @@ public void onClick(View v) {
     }
 
     public void playAudio() {
-        final String mimeType = FileUtils.getMimeType(message.getFileMetas().getName());
+        final String mimeType;
+        if(message.getFileMetas() != null) {
```

Comment:
FileMeta can be non-null but getName() can still be null, it will throw an exception again